overview/valid-constraints/README.md

```diff
@@ -575,7 +575,7 @@ myField = { type : "xml" }
 
 ## udf
 
-The field value, the target object, and an empty metadata structure will be passed to the declared closure/lambda to use for validation. The UDF must return **boolean**, `validate( value, target, metadata ):boolean`
+The field value, the target object, and an empty metadata structure will be passed to the declared closure/lambda to use for validation. The UDF must return **boolean**, `validate( value, target, metadata ):boolean`. NOTE: The target object passed in is actually an instance of "GenericObject", not a struct. To access the underlying struct, use the getMemento() function and perfom any comparisons on that. See the example below. 
```

Comment:
It's important to note that `GenericObject` is ONLY used if the incoming validation target is a structure.  If you use normal objects, it will be that object.  So I think you need to expand this note, to clarify its ONLY when validating structures.